perl-lib/OESS/t/MPLS-Discovery-Interface.t

```diff
@@ -58,6 +58,23 @@ my $res = $interface_discovery->process_results( node => $example_node, interfac
 
 ok($res == 1, "Interface processing reports success");
 
+my @results;
+foreach my $intf (@$example_data) {
+    my $intf_id = $db->get_interface_id_by_names(
+	node => $example_node,
+	interface => $intf->{'name'}
+	);
+    my $result = $db->get_interface(interface_id => $intf_id);
+    if ($result->{'operational_state'} eq $intf->{'operational_state'}) {
+	push(@results, "ok");
+    }
+}
+ok(scalar @results == 3, "DB matches test data");
+
+@$example_data[0]->{'name'} = "fooooo!";
+$res = $interface_discovery->process_results( node => $example_node, interfaces => $example_data );
+is($res, undef, "nonexistent interface fails");
+
 #TODO:
```

Comment:
For our tests here can we add a few tests that will set the operational state to up and then verify that the operational state is now up... and also add an additional interface and verify it populates properly